target/linux/ramips/dts/mt7628an_hiwifi_hc5761a.dts

```diff
@@ -32,11 +32,22 @@
 			linux,default-trigger = "phy1tpt";
 		};
 	};
+
+	gpio_export {
+		compatible = "gpio-export";
+		#size-cells = <0>;
+
+		usbpower {
+			gpio-export,name = "usb";
+			gpio-export,output = <1>;
+			gpios = <&gpio1 7 GPIO_ACTIVE_HIGH>;
+		};
+	};
 };
 
 &state_default {
 	gpio {
-		ralink,group = "i2c", "refclk", "wdt", "p3led_an", "wled_an";
+		ralink,group = "i2c", "refclk", "wdt", "p2led_an", "p3led_an", "wled_an";
```

Comment:
p4led_an?

Reply:
Err, gpio1 7 should be 39? Or is it gpio1 9 = 41 as you state in commit message?

Reply:
`GPIO1 7` is 41.

Reply:
So, gpio1 0 is 34?

Reply:
Sorry, it should be 9, my bad :/

Reply:
Did you retest with the DTS changes? Or just tried 41 on device?